src/index.js

```diff
@@ -2,6 +2,52 @@
  * Returns a very important number
  * @return {number}
  */
-export function myModule() {
-  return 42;
+
+export function lorentzian(options = {}) {
+  const { factor = 8, width = 1000 } = options;
+  const halfWidth = width / 2;
+  const center = factor * halfWidth;
+  const vector = [];
```

Comment:
Please use Float64Array (therefore you also need to fix the length). It will be much faster to create but also to use later. In general when we know the size we should always use Float64Array and all our packages should be compatible with them.